src/XamlStyler/DocumentManipulation/NodeCollection.cs

```diff
@@ -1,9 +1,10 @@
-// � Xavalon. All rights reserved.
+// ? Xavalon. All rights reserved.
```

Comment:
Something weird has been going on with this.

Reply:
should be all fixed in new commit